packages/callhierarchy/src/browser/callhierarchy-tree/callhierarchy-tree-widget.tsx

```diff
@@ -178,9 +190,9 @@ export class CallHierarchyTreeWidget extends TreeWidget {
     private doOpenEditor(uri: DocumentUri, range: Range, keepFocus: boolean): void {
         this.editorManager.open(
             new URI(uri), {
-                mode: keepFocus ? 'reveal' : 'activate',
-                selection: range
-            }
+            mode: keepFocus ? 'reveal' : 'activate',
+            selection: range
+        }
```

Comment:
Nitpick: The formatting seems off to me. I think the opening brace belongs to the next line for TypeScript to format this correctly.

Reply:
I have made the appropriate fix.